cmd/migrate_from_qdrant.go

```diff
@@ -539,25 +538,26 @@ func (r *MigrateFromQdrantCmd) getStartOffset(ctx context.Context, sourceClient
 	return nil, 0, nil
 }
 
-func (r *MigrateFromQdrantCmd) StoreOffset(ctx context.Context, sourceClient *qdrant.Client, sourceCollection string, offset *qdrant.PointId, offsetCount uint64) error {
+func (r *MigrateFromQdrantCmd) StoreStartOffset(ctx context.Context, targetClient *qdrant.Client, sourceCollection string, offset *qdrant.PointId, offsetCount uint64) error {
```

Comment:
The reader function is called `GetStartOffset`. So renamed this to `StoreStartOffset` instead of `StoreOffset` for consistency.

Reply:
https://go.dev/doc/effective_go#Getters

`SetStartOffset` would be more idiomatic.